R/pkg/R/SQLContext.R

```diff
@@ -462,14 +462,19 @@ read.df <- function(sqlContext, path = NULL, source = NULL, ...) {
     source <- callJMethod(sqlContext, "getConf", "spark.sql.sources.default",
                           "org.apache.spark.sql.parquet")
   }
-  sdf <- callJMethod(sqlContext, "load", source, options)
```

Comment:
Hmm - yes. I could add something to `SQLUtils` in `r/api` I guess. But in general why did we need to deprecate the `load()` functions ?